doc/FAQ.md

```diff
@@ -49,6 +51,17 @@ For AliceO2 libraries see the next question.
 
 Add the library name to the list `O2_LIBRARIES_NAMES` in [FindAliceO2.cmake](../cmake/FindAliceO2.cmake)
 
+## Run 
+
+### Why are my QC processes using 100% CPU ? 
+
+When running `o2-qc` or other qc binaries, the system will show that the processes use 100% of the CPU. This is due to the DPL default rate for data source devices. 
```

Comment:
not only data sources, devices with inputs also have an event loop which runs with this rate